docs/generator/requirements.txt

```diff
@@ -1,3 +1,3 @@
 mkdocs>=1.0.1
 mkdocs-material
-
+pymdown-extensions
```

Comment:
How does it relate to the issue which this PR is trying to solve?

Reply:
Removed, no longer needed.